### ISSUE TITLE
[accessibility-quiz]: step-56 fails valid code

### ISSUE THREAD

**Issue by ShaunSHamilton** [freeCodeCamp/freeCodeCamp#48182]
### Describe the Issue

**CORRECTION**

As described in the below forum post, the tests should account for the outcome of the code. Not just look at the specific StyleRule

### Affected Page

https://www.freecodecamp.org/learn/2022/responsive-web-design/learn-accessibility-by-building-a-quiz/step-56

### Code

```css
.info input, .info label {
  display: inline-block;
  text-align: right;
}


.info input {
  width: 50%;
  text-align: left;
}

.info label {
  width: 10%;
  min-width: 55px;
}
```

### Expected Behaviour

The tests should be changed to allow for valid code to pass. The most robust way of ensuring this is probably by comparing the computed values for the elements.

https://forum.freecodecamp.org/t/learn-accessibility-by-building-a-quiz-step-56/565245

---

An issue with the https://github.com/freeCodeCamp/freeCodeCamp/labels/help%20wanted label is open for contribution. The first comprehensive PR created will be reviewed and merged. We typically do not assign issues to anyone other than long-time contributors.

If you would like to contribute, and have not read the contributors docs, please do so here: https://contribute.freecodecamp.org/#/

If you have any issues with contributing, be sure to join us on the [contributors channel](https://discord.com/channels/692816967895220344/715074489422970962), or on the [contributors sub-forum](https://forum.freecodecamp.org/c/contributors/3)

**Comment by hbar1st** [comment]
I think Randell's point about this step was that putting the text-align: right in the .info input,.info label rule *should* also be accepted because it ultimately gets overriden by the .info input selector following it. (so the inputs stay on the left and the labels on the right)

(so it wouldn't be enough to change the description of the exercise)

**Comment by ShaunSHamilton** [comment]
@hbar1st Thank you for correcting. I have updated the original post.

**Comment by nayabatir1** [comment]
I'm just curious, isn't it bad practice to change same css properties twice. As it would hard to debug when working with real projects.

**Comment by ShaunSHamilton** [comment]
@nayabatir1 Often, it might be. However, it is very common practice. More so, within the same style definition:

```css
.example {
  margin: 0;
  margin-top: 10px;
}
```

Our main goal here is to allow correct code to pass. It might not be _best practice_, but it does achieve the given user story.

If we have lessons covering best practices, then we can/will/do enforce the code used/(not used).
